lib/turbo/engine.rb

```diff
@@ -15,6 +15,27 @@ class Engine < Rails::Engine
       #{root}/app/jobs
     )
 
+    # If the parent application does not use Active Job, app/jobs cannot
+    # be eager loaded, because it references the ActiveJob constant.
+    #
+    # When turbo-rails depends on Rails 7 or above, the entire block can be
+    # reduced to
+    #
+    #   unless defined?(ActiveJob)
+    #     Rails.autoloaders.once.do_not_eager_load("#{root}/app/jobs")
+    #   end
+    #
+    initializer "turbo.no_active_job", before: :set_eager_load_paths do
+      unless defined?(ActiveJob)
```

Comment:
Would this work instead?

`ActiveSupport.on_load :active_job do`

Reply:
I don't know. I'm following the pattern used in the Action Cable check below.

Reply:
Correct, sorry I was thinking of the opposite case but this initializer occurs in the absence of AJ :pray: